src/Data/FunctorF.purs

```diff
@@ -0,0 +1,17 @@
+module Data.FunctorF where
+
+import Data.NaturalTransformation (type (~>))
+
+-- | A type class for higher order functors; types of kind `(* -> *) -> * -> *`.
```

Comment:
I figured including the kind would disambiguate which kind of HO functor we were talking about. I also wasn't sure if it was the functor category exactly, but the latter makes sense!